src/pages/room/index.ts

```diff
@@ -1,5 +1,5 @@
 import RoomList from "./list";
 import RoomDetail from "./detail";
-import Jebaited from "./jebaited";
+import MyRooms from "./myrooms";
```

Comment:
`./myrooms` directory doesn't exist. Please include in the commit.

Reply:
renamed jebaited to myrooms